build.gradle

```diff
@@ -113,7 +113,10 @@ project('solr-hadoop-core') {
       transitive = false
     }
     compile("org.apache.solr:solr-commons-csv:3.5.0")
-    compile("org.apache.mahout:mahout-core:${mahoutVersion}") {
+    compile("org.apache.mahout:mahout-mr:${mahoutVersion}") {
```

Comment:
In Mahout versions > 0.10.0, mahout-core has been split into these two dependencies.

Reply:
nice!